blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/ProfileApiTest.java

```diff
@@ -222,50 +222,27 @@ public void createUserFavouriteFolderTest() throws Exception {
         Assert.assertEquals(0, l.size());
 
 
-        map = new RequestBuilder(baseUrl)
+        new RequestBuilder(baseUrl)
             .put("/organizations/jenkins/pipelines/folder1/favorite/")
             .jwtToken(token)
             .data(ImmutableMap.of("favorite", true))
+            .status(405)
             .build(Map.class);
 
-        validateFolder(folder1, (Map) map.get("item"));
-        l = new RequestBuilder(baseUrl)
-            .get("/users/"+user.getId()+"/favorites/")
-            .jwtToken(token)
-            .build(List.class);
-
-        Assert.assertEquals(1, l.size());
-        Map folder = (Map)((Map)l.get(0)).get("item");
-
-        validateFolder(folder1, folder);
-
-        href = getHrefFromLinks((Map)l.get(0),"self");
-
-        Assert.assertEquals("/blue/rest/organizations/jenkins/pipelines/folder1/favorite/", href);
-
-        map = new RequestBuilder(baseUrl)
-            .put(href.substring("/blue/rest".length()))
```

Comment:
Nothing to be tested here anymore?

Something I've been asking people to do is add more comments in tests. Taking this test for example ... I've no real idea what the test is expecting to happen and why ... multiple calls to create instances of `RequestBuilder` and I've no real idea why and what's expected to happen.